README.md

```diff
@@ -1,2 +1,119 @@
-# spire-mysql-demo
-Demonstration of using SPIRE-issued X.509-SVIDs to set up an authenticated mTLS connection to a MySQL database
+# MySQL mTLS Authentication using SPIRE
+
+Demonstration of using SPIRE-issued X.509-SVIDs in MySQL database and clients, providing client authentication
+and connection encryption in MySQL via mutual-TLS (mTLS), deployed on Kubernetes. This project assumes readers to have 
+basic knowledge of the following technologies and concepts:
+
+1. [SPIFFE](https://spiffe.io/) and [SPIRE](https://github.com/spiffe/spire)
+2. [Kubernetes](https://kubernetes.io/)
```

Comment:
nit: starting each numbered bullet with `1. ` rather than the next number is a little easier to maintain